pages/addPlant.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import Form from "@/components/Form";
+import styled from "styled-components";
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  background-color: white;
+  padding: 2px;
+`;
+
+export default function AddPlant({ handleAddPlant }) {
+  function entrySucessful() {
+    return alert("Plant added sucessfully");
+  }
```

Comment:
you don't need to write an extra function here, you can add it to the handleSubmit function in the Form component :)
```suggestion
```

Reply:
we know, this is only for the mvp now. Later we will updated the function.